artist/forms.py

```diff
@@ -22,16 +22,39 @@ class ArtistApplyForm(forms.Form):
         )
     )
     campaign_reason = forms.CharField(
-        label="Why are you raising money?",
+        label="What are you raising money for?",
         widget=forms.Textarea(
             attrs={"placeholder": "We are trying to record our album..."}
         ),
     )
-    campaign_expenses = forms.CharField(
-        label="What do you need the money for?",
-        widget=forms.Textarea(
-            attrs={"placeholder": "Mixing, mastering, studio time, etc..."}
-        ),
+    amount_raising = forms.CharField(
+        label="Amount Raising", widget=forms.TextInput(attrs={"placeholder": "$1,000"})
+    )
+    giving_back = forms.CharField(
+        label="% Back To Investors",
+        widget=forms.TextInput(attrs={"placeholder": "50%"}),
+    )
+    campaign_start = forms.CharField(
+        label="Campaign Start Date",
+        widget=forms.TextInput(attrs={"placeholder": "1/17/20"}),
+    )
```

Comment:
You could use a `DateField` instead, which could then be rendered as a calendar widget in the frontend.

Reply:
Yeah, that would be awesome. It looks like foundation has a date picker. Tried adding it a few different ways but can't seem to get it working. Is there a simple way to add a widget to the form?

Reply:
After taking a look, this is going to be slightly more work than I thought, so probably not worth doing right now. I think it's still worth using a `DateField` though, so that the date entered is validated.